config/7.1.0/securebanking/ig/scripts/groovy/ProcessDetachedSig.groovy

```diff
@@ -232,7 +234,7 @@ def isJwsSignatureValid(String detachedSignatureValue, RSAPublicKey rsaPublicKey
     boolean criticalParamsValid = validateCriticalParameters(jwsHeader);
     if (!criticalParamsValid) {
         logger.error(SCRIPT_NAME + "Critical params validations failed. Stopping further validations.")
-        return newResultPromise(false)
+        return false
```

Comment:
very subtle bug, something to be careful with.

Due to groovy true: https://groovy-lang.org/semantics.html#the-groovy-truth , Promise<Boolean>(false) evaluates to true